pagespeed/automatic/proxy_fetch.h

```diff
@@ -342,7 +342,7 @@ class ProxyFetch : public SharedAsyncFetch {
   //
   // This is used only for testing.
   static const int kTestSignalTimeoutMs = 200;
-
+  void SetTrustedInput() { trusted_input_ = true; }
```

Comment:
Sorry... One more thing....And it should take the new value as a formal.
On Oct 27, 2015 12:22 PM, "Otto van der Schaaf" <notifications@github.com>
wrote:

> In pagespeed/automatic/proxy_fetch.h
> <https://github.com/pagespeed/mod_pagespeed/pull/1161#discussion_r43145911>
> :
>
> > @@ -342,7 +342,7 @@ class ProxyFetch : public SharedAsyncFetch {
> >    //
> >    // This is used only for testing.
> >    static const int kTestSignalTimeoutMs = 200;
> > -
> > +  void SetTrustedInput() { trusted_input_ = true; }
>
> @jmarantz <https://github.com/jmarantz> re: "nit: style guide says to
> spell this 'set_trusted_input'."
> renamed accordingly
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pagespeed/mod_pagespeed/pull/1161/files#r43145911>.
>